docs/README-agent-install.md

```diff
@@ -24,18 +24,21 @@ This document describes Puppet agent installation and setup on Cisco Nexus switc
 #### Platform and Software Minimum Requirements
 
 * The Cisco NX-OS Puppet implementation requires open source Puppet version 4.0 or Puppet Enterprise 2015.2
+* The Cisco IOS XR Puppet implementation requires open source Puppet version 4.3.2 or Puppet Enterprise 2015.3.2
```

Comment:
Note: our `metadata.json` does not reflect the same Puppet Enterprise version requirement.

```
  "requirements": [
    {
      "name": "pe",
      "version_requirement": "2015.2.x"
    },
```